installer/pkg/cmd/gcp_broker.go

```diff
@@ -443,3 +450,54 @@ func removeConfigs(dir string, filenames []string) error {
 	}
 	return nil
 }
+
+type createBrokerConfig struct {
+	name  string // name of the broker
+	title string // title of the broker
+}
+
+// NewCreateGCPBrokerCmd returns a cobra command which creates a new GCP service
+// broker without adding it to the existing Kubernetes cluster.
+func NewCreateGCPBrokerCmd() *cobra.Command {
+	cfg := &createBrokerConfig{
+		name:  "default",
+		title: "Default Broker",
+	}
+	cmd := &cobra.Command{
+		Use:   "create-gcp-broker",
+		Short: "Create a GCP broker",
+		Long:  "Creates a GCP broker without adding it to an existing cluster",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createGCPBroker(cfg)
+		},
+	}
+	cmd.Flags().StringVar(&cfg.name, "name", "default", "Broker name, lowercase, hyphens allowed")
```

Comment:
Maybe we want to make this one required and change it to a different default value? It is highly possible that the name "default" is already taken and getOrCreateVirtualBroker() will silently cover the 409 error.

Reply:
That's intentional. Goal is that by default behavior user gets the same default broker but they can override if they need.
The function already handles "already exists" correctly.